samples/coop_synchronization/sample.yaml

```diff
@@ -0,0 +1,37 @@
+sample:
+  description: A simple application that demonstrates even more basic sanity of the kernel
+    compared to samples/synchronization. The main variant is the polled_uart which shouldn't
+    go through any interrupt driven context switch in a no-systick scenario.
+  name: Coop Synchronization Sample
+tests:
+  sample.kernel.coop_synchronization.polled_uart:
+    build_on_all: true
```

Comment:
please remove this, this is not intended for general usage.

Reply:
Hi @nashif , may I ask for a re-review of this change? I've removed it.